src/envoy/http/authn/filter_context.cc

```diff
@@ -71,6 +72,29 @@ void FilterContext::setPrincipal(const iaapi::PrincipalBinding& binding) {
   }
 }
 
+bool FilterContext::getJwtPayload(const std::string& issuer,
+                                  std::string* payload) const {
+  const auto filter_it =
+      dynamic_metadata_.filter_metadata().find(Utils::IstioFilterName::kJwt);
```

Comment:
Is this dependency data structure documented some where?  Is it already implemented by JWT filter?

Reply:
It's implemented by (proxy) JWT filter. For upstream, I expect to have API defined in Envoy. Wdyt?